lib/validators/csp_script_src_validator.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class CspScriptSrcValidator
+  def initialize(opts = {})
+    @opts = opts
+  end
+
+  def valid_value?(values)
+    return true if values == ""
```

Comment:
This can be optional, as `[].all?` will return `true` (so with `values` being blank, the return value will be `true`) :slightly_smiling_face: